### ISSUE TITLE
add button

### ISSUE THREAD

**Issue by Yetreyert** [makerbase-mks/MKS-TFT#521]
is it possible to add the auto level button to the manual level screen? has 2 free spaces to use

**Comment by mks-viva** [comment]
Hello! Usually only one leveling method is used, and the user does the same. Because the user is afraid of misoperation, the leveling will fail. Of course, we will collect more suggestions.

**Comment by Yetreyert** [comment]
It depends. If I change the plate you must first align it well and then I create the mesh or start with self-leveling of the bltouch. I think it is essential as a thing to not always depend on the PC. It is a request that was also made by the entire Italian community. I'm trying to understand how the various pages have been set up also to add an on and off lights command

**Comment by mks-viva** [comment]
Use MKS TFT35? 
You can download the latest version firmware and update it
https://github.com/makerbase-mks/MKS-TFT/tree/master/MKS-TFT3.5

Change leveling mode: Settings->Config->Manchine->Leveling settings(click next)->Leveling settings->Enable auto leveling (YSE)

**Comment by Yetreyert** [comment]
Yes v2 with mks robin nano 2.0. Unfortunately I also saw that there is a problem with the neopixel library too bad

**Comment by Yetreyert** [comment]
Yes v2 with mks robin nano 2.0. Unfortunately I also saw that there is a problem with the neopixel library too bad
--
Inviato da Libero Mail per Android mercoledì, 24 febbraio 2021, 11:18AM +01:00 da mks-viva  notifications@github.com :
